token.temp.py

```diff
@@ -1 +1,2 @@
-TOKEN = ''
\ No newline at end of file
+BOT_TOKEN = '160983951:AAFVG4cvpbVfuYF8NVUFfxfQr3zmLlxvb1U'
+TEXT_TOKEN = "22e74d52026ed99a6aef6e07d77a6b73cc9b799a98ddcf4b69793b4b"
```

Comment:
Please don't commit the token, Mention document of how to retrieve these keys so that other users can add the tokens here. Good job converting TOKEN to BOT_TOKEN